quickstart.md

```diff
@@ -0,0 +1,136 @@
+# Ceramic Playground Quickstart (& Video Reference)
+## Getting Started
+1. Click on the "Connect Wallet" Button, if you don't have a wallet MetaMask is a good place to start.
+2. Follow the prompts that are in the upper right hand corner to authenticate with 3ID-Connect. 
+   - After you authenticate open your web browser tools, you should see something like: `Connected with DID: did:3:kjz...r5y`. If you do not see this please reach out in Discord.
+
+## Basic IDX Usage: 
+Let's create a basic profile, this is a default schema so it's easy. All of the following commands can be completed in the web brower's console.
+
+1. `await idx.get('basicProfile', ceramic.did.id)`
+  - IDX.get has 1 mandatory parameter that is a string, usually it isn't so readable. This is readable because it's been aliased. For more information see: https://developers.idx.xyz/build/aliases/
+  - Then we need to "sign" the request, in the Playground we access it from the Ceramic instance. It's generated when you authenticate with 3ID-Connect.
+  - OUTPUT: ` `
+   - There will be nothing here because there's no profile attached to the DID.
+2. `await idx.set('basicProfile', { name: 'Your Name', description: 'Tell us about yourself!' emoji:'✌🏻' })`
+  - Notice that we did not provide our DID, but it's updating our record. This is because we default to the currently active ID.
+  - These fields are optional, but they can be set to be required by the schema.
+3. `await idx.merge('basicProfile', { emoji: '👍🏻'})`
+   - Updating records is easy, change any details you need to anytime you need to.
+
+## Creating a Custom Schema:
+Schema creation might seem daunting at first, but it's quite simple when you understand how it's all built. 
+
+### Helper Functions:
+To start we will need to create a few helper functions. These are all included in our `@ceramicstudio/idx-tools` package, this is just to demystify them. To use them you can just copy & paste the code into your browser console.
```

Comment:
These are internal functions in IDX tools, I'd rather we avoid using/referencing them elsewhere please as they are not documented and could lead to more confusion.
The tile creation can be simply `const doc = await TileDocument.create(ceramic, content, metadata)`, the rest are sanity checks.

Reply:
This file shouldn't be here. Totally my mistake for including it. This is more for the "quickstart" documentation. I need to do some testing to make sure that things behave properly with the IDX tools and whatnot.